### ISSUE TITLE
[Feature] Feature(bar3D): add startValue option to specify where the bar3D start

### ISSUE THREAD

**Issue by webzhou** [apache/echarts#20139]
### What problem does this feature solve?

![微信截图_20240710164607](https://github.com/apache/echarts/assets/11323836/1240047f-24fc-4d2c-ac6a-d5edb92393d9)
![微信截图_20240710164556](https://github.com/apache/echarts/assets/11323836/c5132ea6-5edf-45df-a21a-687ebde153de)

### What does the proposed API look like?

 add startValue option to specify where the bar3D start，now if has 0，bar3D startValue is 0, else startValue is min

**Comment by plainheart** [comment]
如果这个和 #20138 属于一个问题，请整合一下并关闭其中一个。

**Comment by webzhou** [comment]
> 如果这个和 #20138 属于一个问题，请整合一下并关闭其中一个。

是的 是一个问题，希望 bar3D 可以设置一个startValue，像bar 那样，如果 z轴中出现 0，开始的值就会从0开始，但是实际效果不想从0 开始

**Comment by helgasoft** [comment]
Possible workaround with _line3D_ - [Demo](https://echarts.apache.org/examples/en/editor.html?gl=1&code=EYEwhgLmAEC80G0BQ1EEYA01PYLoZUQAYA6AVi1IugGZclcBuJAehegAcAnAU24HsAxjwDOIgJYA7AObRgYLjQAi0cFGhSI_aABspPZashgkIqFwgAtAG5gdAVx5xoAWhrMdamPARMkoYxIAWzAOAAowwUEASjgAPmgAb0IogCZnBCiEInxoLLRcswUrWwceP1RPQI57EQALMMzBVKwo3GjmAF8O0x4ucVFnKqhg0IiADyxxWNgE5NReCHsuSSTCVAgATw4eAC5oAHI9SQMlA4JUVC99yfXoSTAgvcPjngPoAGoNC8vXgGUtjpnoloAB3cQgCB1fYANla_B0_C4-wONS4HCB706P1QPCCHDqYAkIn2IPEEDxAM2QNJeQRSJRwDKWOgnUInS6PV6_UG8BEfQGIhIgn4kkEkDC82gWx2jIUynOhAeTzlXEVV2M-wCUB-eIJRPEJKSGgpQSpNONIsRyMOTMcLOxhHJlMBwOg_A4YEE5M2-yorKQ3WYSA9EHEoucUq0CLDHFJjsqPGkPEkIHjP3GAEFxoblOnCJts7mlPnUAAvIsiPNJBPQaT9EDVxK1_k8kSBxhAA).
However, this workaround is affected by a bug - vertical 3D lines show [no tooltips](https://github.com/ecomfe/echarts-gl/issues/528).

**Comment by webzhou** [comment]
3d柱状图（3dbar），如果z轴包含0，是否支持从最小值开始渲染，现在3d柱状图都是从0开始画的，可否像 [bar] 一样，添加 startValue 配置项，使起始点可以自定义
